iosApp/iosApp/Localizable.xcstrings

```diff
@@ -14478,7 +14572,7 @@
         },
         "fr" : {
           "stringUnit" : {
-            "state" : "translated",
+            "state" : "needs_review",
             "value" : "Envoyer des commentaires sur l’application"
```

Comment:
Xcode was telling me French was 93% translated but the other languages were all only 92%, and I knew that couldn’t be right. It looks like when this string was changed from `"Send app feedback"` to `"Send App Feedback"` in #895, the French translation was not reset to needs review like the rest of the translations were.